src/components/App.tsx

```diff
@@ -4,12 +4,22 @@ import Header from "./Header/Header";
 import MovieDetails from "./MovieDetails/MovieDetails";
 import { useAppSelector } from "../redux/hooks";
 import { selectMovie } from "../redux/movieSlice";
+import { useLocation, useNavigate} from "react-router-dom";
+import { useEffect } from "react";
 
 import styles from "./app.module.css";
 
 function App() {
+  const location = useLocation();
+  const navigate = useNavigate();
   const movie = useAppSelector(selectMovie);
 
+  useEffect(() => {
+    if (location.pathname === "/") {
+      navigate("/search");
+    }
+  }, [location.pathname, navigate]);
```

Comment:
Do you really need this for redirecting?

Reply:
but task requirements say to redirect to "/search" when path is "/" or is there another better way

Reply:
Definitely we need a "search" route

Let's see what you have indicated in the routes